### ISSUE TITLE
'Save As' should place the original/closed file into the Recent files list

### ISSUE THREAD

**Issue by trush44** [notepad-plus-plus/notepad-plus-plus#5857]
### Description of the Issue
After performing a 'Save As' on a file, NP++ should place the original/closed file into the Recent files list.

### Steps to Reproduce the Issue
1. Open a file in NP++
2. Closed the file
       -notice the closed file is added to the list of Recent files in the 'File' menu (as expected)
3. Open the file again
4. Do a 'Save As' some other file name
       -notice the original/closed file is NOT added to the list of Recent files

### Expected Behavior
The original file should be added to the list of Recent files once it has been closed out via 'Save As'.

### Actual Behavior
The original file is not added to the Recent files list.

**Comment by sasumner** [comment]
It's a GOOD idea.  I presume you want an easy way to continue working with BOTH files.  For this case, I do "Save a Copy As..." and then, with the original file active (because it is still open), I do "File -> Open..." which will open a file dialog in the same folder as the original file (99% of the time I have both files in the same folder) and then it is easy to find and open the secondary file (and thus have both open to continue working with both).

Actually, I lied. I used to do that.  Now I have a script called "Save As Copy And Then Open The Copy" that makes the process even simpler. :)

**Comment by trush44** [comment]
> 
> 
> It's a GOOD idea. I presume you want an easy way to continue working with BOTH files. For this case, I do "Save a Copy As..." and then, with the original file active (because it is still open), I do "File -> Open..." which will open a file dialog in the same folder as the original file (99% of the time I have both files in the same folder) and then it is easy to find and open the secondary file (and thus have both open to continue working with both).
> 
> Actually, I lied. I used to do that. Now I have a script called "Save As Copy And Then Open The Copy" that makes the process even simpler. :)

Yep, that is the exact reason. Thanks for the tip on 'Save a Copy As'. A lot of times though the Save As folder isn't the same.

**Comment by sasumner** [comment]
BTW I think it really helps the developers here when you expound on WHY you want something.  Someone who at first might say "meh" to a simple "I want this." and move on to something else might actually grab onto an issue when the true value is explained ("I want this because it helps me do this and that and...").  I think that in tandem we have now covered that for this issue. :)

**Comment by alankilborn** [comment]
This was fixed and released in N++ 8.2.1.
The fix was in addressing #10946.
Recommend CLOSURE of this item.
